Makefile

```diff
@@ -62,26 +62,37 @@ deploy-via-kustomize: manifests $(KUSTOMIZE)
 deploy: $(SKAFFOLD) $(HELM)
 	$(SKAFFOLD) run -m etcd-druid --kubeconfig=$(KUBECONFIG_PATH)
 
+.PHONY: deploy-dev
+deploy-dev: $(SKAFFOLD) $(HELM)
+	$(SKAFFOLD) dev -m etcd-druid --kubeconfig=$(KUBECONFIG_PATH)
+
+.PHONY: deploy-debug
+deploy-debug: $(SKAFFOLD) $(HELM)
+	$(SKAFFOLD) debug -m etcd-druid --kubeconfig=$(KUBECONFIG_PATH)
+
+.PHONY: undeploy
+undeploy: $(SKAFFOLD) $(HELM)
+	$(SKAFFOLD) delete -m etcd-druid --kubeconfig=$(KUBECONFIG_PATH)
+
```

Comment:
For a cleaner approach and to minimize repetition, could we consider setting `--kubeconfig=$(KUBECONFIG_PATH)` globally, rather than specifying it for each target? We could set it once and export it for multiple targets, similar to https://github.com/gardener/etcd-druid/blob/2102a45291ffad1f415d631521617fc5d189c4a7/Makefile#L34

Reply:
Thanks for the suggestion. Incorporated in https://github.com/gardener/etcd-druid/pull/777/commits/edb735ab6f1ef0aa2d11270f36d20eacf7b117e8